src/cswinrt/code_writers.h

```diff
@@ -4580,44 +4329,79 @@ event % %;)",
                 w.write_temp("__params[%]", param_index);
         }
 
+        void write_convert_to_managed_function(writer& w) const
+        {
+            if (interop_dll_type != "")
+            {
+                w.write(R"(
+[UnsafeAccessor(UnsafeAccessorKind.StaticMethod, Name="ConvertToManaged")]
```

Comment:
Nit: formatting (same below)
```suggestion
[UnsafeAccessor(UnsafeAccessorKind.StaticMethod, Name = "ConvertToManaged")]
```